plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -364,14 +364,16 @@ public void verifyText(VerifyEvent event) {
       if (MavenCoordinatesValidator.validateGroupId(newGroupId.trim())) {
         String newSuggestion = suggestPackageName(newGroupId);
         updatePackageField(newSuggestion);
-        previousSuggestion = newSuggestion;
+      } else if (newGroupId.trim().isEmpty()) {
+        updatePackageField("");
       }
     }
 
     private void updatePackageField(String newSuggestion) {
       if (getPackageName().isEmpty() || getPackageName().equals(previousSuggestion)) {
         javaPackageField.setText(newSuggestion);
       }
+      previousSuggestion = newSuggestion;
```

Comment:
Just saw this. This was an intended behavior. If the suggested package name based on the group ID becomes to match what is entered in the package field, we go back into the "tracking" mode, in which we continue to auto-fill the package field based on group ID changes. If at any point a user manually modifies the package field and the value becomes different from suggested package based on the group ID, we go into the "detached" mode, and we stop updating the package field. Then, if a user changes her mind to modify the package to match the group ID, then we go back to the "tracking" mode.

So, this is the current behavior. What else could you suggest for updating the package?